nservicebus/definition-event.include.md

```diff
@@ -0,0 +1 @@
+An [**event**](/nservicebus/messaging/messages-events-commands.md#event) is a message that is used to communicate that some action has taken place. It is published from a single logical sender, and is processed by (potentially) many receivers.
```

Comment:
>and is processed by (potentially) many receivers.

remove the parenthesis for potentially. adds no value. harder to read.  Also change receivers to endpoints.